ibtk/src/utilities/StandardTagAndInitStrategySet.cpp

```diff
@@ -56,13 +56,8 @@ namespace IBTK
 
 StandardTagAndInitStrategySet::~StandardTagAndInitStrategySet()
 {
-    if (d_managed)
-    {
-        for (const auto& strategy : d_strategy_set)
-        {
-            delete strategy;
-        }
-    }
+    if (!d_managed)
+        for (auto& strategy : d_strategy_set) strategy.release();
     return;
 } // ~StandardTagAndInitStrategySet
```

Comment:
What is going on here? Won't the calls to `release` create memory leaks? I do not understand why we need this destructor.

Reply:
In the current interface, the caller gets to decide if they hand over management of the provided objects.